criu/sk-unix.c

```diff
@@ -2225,8 +2225,8 @@ int add_fake_unix_queuers(void)
 	list_for_each_entry(ui, &unix_sockets, list) {
 		if ((ui->ue->uflags & (USK_EXTERN | USK_CALLBACK)) || ui->queuer)
 			continue;
-		if (!(ui->ue->state == TCP_ESTABLISHED && !ui->peer) &&
-		     ui->ue->type != SOCK_DGRAM)
+		if (!(ui->ue->type == SOCK_STREAM && ui->ue->state == TCP_ESTABLISHED && !ui->peer) &&
+		    !(ui->ue->type == SOCK_DGRAM  && !ui->peer))
```

Comment:
Where is SOCK_SEQPACKET?
do we need to add a fake queuer if there is nothing in a recv queue?

Reply:
> Where is SOCK_SEQPACKET?
note: we don't support established non-stream: https://github.com/checkpoint-restore/criu/blob/criu-dev/criu/sk-unix.c#L1828